### PR TITLE
Fixes configuration bug in notifier, kafka lifecycle and backfills tests

### PR DESCRIPTION
**Pull request by adriancole**
Typo resulted in an invalid webhook url. Also, the Kafka bootstrap
wasn't closing properly. Finally, there were no integration tests.

### CODE REVIEW

File: notifier/src/main/java/com/expedia/adaptivealerting/notifier/config/AppConfig.java [hunk 1]
```diff
@@ -74,7 +81,7 @@ public String getKafkaTopic() {
         return this.topic;
     }
 
-    @Value(("webhook.url"))
+    @Value("${webhook.url}")
```
Comment (adriancole) [inline notifier/src/main/java/com/expedia/adaptivealerting/notifier/config/AppConfig.java]: this was the typo

Comment (williewheeler) [reply notifier/src/main/java/com/expedia/adaptivealerting/notifier/config/AppConfig.java]: ack, thanks

**Comment by williewheeler** [comment]
Merged, but I had a minor request regarding externalizing versions. Thanks for the PR @adriancole.

File: notifier/pom.xml [hunk 2]
```diff
@@ -51,6 +51,26 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>mockwebserver</artifactId>
+            <version>3.11.0</version>
```
[the remark this answers is not in the document]
Comment (adriancole) [reply notifier/pom.xml]: cool.. do you mind if I refactor the convention? In zipkin, we've had some success defaulting to the primary artifact id + .version. Ex `maven-surefire-plugin.version` not  `maven.surefire.plugin.version` as it makes search easier. By primary I mean some artifacts are subordinate to others and pinned at same version. Ex. `maven-surefire-plugin.version` would be used also for maven-failsafe-plugin in the same way `okhttp.version` would for mockwebserver. This prevents some degree of misalignments.

[the remark this answers is not in the document]
Comment (williewheeler) [reply notifier/pom.xml]: Yeah, go for it. That sounds great.

**Pull request merged**